includes/class-sensei-course-progression-settings.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * File containing the Sensei_Course_Progression_Settings class.
+ *
+ * @package sensei-lms
+ * @since 4.0.0
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Adds the course progression settings to the course.
+ *
+ * @since 4.0.0
+ */
+class Sensei_Course_Progression_Settings {
+
+	const VIDEO_COURSE_AUTOCOMPLETE = '_video_course_autocomplete';
```

Comment:
Any ideas for better meta field names?

Reply:
Maybe just swap `course` and `video` so that we're moving from the general to the specific (i.e. `_course_video_autocomplete`)? We also usually try to leave a brief comment when defining constants / variables.

Reply:
One more thing I almost forgot about. 🙂 When we add new post meta or options to the database, we also need to add them to the [data cleaner](https://github.com/Automattic/sensei/blob/master/includes/class-sensei-data-cleaner.php#L248) so that when the plugin is uninstalled, we remove plugin-specific data from the database.